src/scripts/restore-latest.sh

```diff
@@ -0,0 +1,22 @@
+restore() {
+    if [ -z "${SKPR_BACKUP}" ]; then
+        SKPR_BACKUP=$(skpr backup list "${SKPR_ENV}" --json | jq .[0].Name)
```

Comment:
Check if you need raw response

Reply:
Checked and nope - doesn't support indexing when using raw.